src/GZCTF/Controllers/ErrorController.cs

```diff
@@ -0,0 +1,16 @@
+using Microsoft.AspNetCore.Mvc;
+
+namespace GZCTF.Controllers;
+
+[ApiController]
+[Route("/error")]
+public class ErrorController : ControllerBase
+{
+    [HttpGet]
+    [HttpPost]
+    [HttpPut]
+    [HttpDelete]
+    [Route("500")]
+    [ProducesResponseType(typeof(RequestResponse), StatusCodes.Status500InternalServerError)]
+    public Task<IActionResult> InternalServerError(CancellationToken cancellationToken) => Task.FromResult<IActionResult>(StatusCode(500, new RequestResponse(GZCTF.Program.StaticLocalizer[nameof(GZCTF.Resources.Program.Identity_DefaultError)], StatusCodes.Status500InternalServerError)));
```

Comment:
This should use the localizer from DI instead of `StaticLocalizer`. You can simply add an `IStringLocalizer<Program>` to the constructor parameters of the controller type so that a localizer instance will be injected by the framework, and then save it as a readonly field for use.